resources/views/admin/employees/index.blade.php

```diff
@@ -1,112 +1,165 @@
-		<li class="breadcrumb-item active">Employees</li>
-	</ol>
-@stop
+@extends('layouts.master')
+
 @section('content')
-    <div class="card">
-        <div class="card-body">
-            <div class="float-right">
-                <select class="form-control" id="filter">
-                    <option value="select">Select Employees</option>
-                    @foreach($filters as $filter)
-                    <option value="{{$filter}}" @if($filter==$selectedFilter) selected @endif>{{ucfirst(trans($filter))}}</option>
-                    @endforeach
-                </select>
+<!-- Breadcrumbs Start -->
+<div class="content-header">
+  <div class="container-fluid">
+    <div class="row mb-2">
+      <div class="col-sm-6">
+        <h1 class="m-0">Employees</h1>
+      </div>
+      <div class="col-sm-6">
+        <ol class="breadcrumb float-sm-right">
+          <li class="breadcrumb-item"><a href="{{ url('employees') }}">People Management</a></li>
+          <li class="breadcrumb-item active">Employees</li>
+        </ol>
+      </div>
+    </div>
+  </div>
+</div>
+<!-- Breadcrumbs End -->
+
+<!-- Error Message Section Start -->
+@if (Session::has('error'))
+    <div class="content">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-12">
+                    <div class="alert alert-danger" align="left">
+                        <a href="#" class="close" data-dismiss="alert">&times;</a>
+                        <strong>Error!</strong> {{Session::get('error')}}
+                    </div>
+                </div>
             </div>
-            <h4 class="card-title"> {{$active_employees}}  Active / {{$employees->count()}} Employees</h4>
-            <div class="table-responsive m-t-40">
-                <table id="myTable" class="display nowrap table table-hover table-striped table-bordered" cellspacing="0" width="100%">
-                    <thead>
-                    <tr>
-                        @if(count($employees) > 0)
-                        <th>Name</th>
-                        <th>Email</th>
-                        <th>Mobile </th>
-                        <th>Designation</th>
-                        <th>Branch</th>
-                        <th>Department</th>
-                        <th>Joining Date</th>
-                        <th>Employment Status</th>
-                        <th>Actions</th>
-                    </tr>
-                    </thead>
-                    <tbody>
-                    @foreach($employees as $employee)
-                    <tr>
-                        <td>{{$employee->firstname}} {{$employee->lastname}}</td>
-                        <td>{{$employee->official_email}}</td>
-                        <td>{{$employee->contact_no}}</td>
-                        <td>{{ ucfirst(trans($employee->designation))}}</td>
-                        {{--<td>{{isset($designations[$employee->designation]) ? $designations[$employee->designation] : ''}}</td>--}}
```

Comment:
@JoharAlam 
Please remove commented code like this.

Reply:
@awaidanjum I will remove it